bp_top/test/tb/bp_top_fpga/bp_fpga.tcl

```diff
@@ -0,0 +1,2599 @@
+#*****************************************************************************************
+# Vivado (TM) v2019.1 (64-bit)
+#
+# bp_fpga.tcl: Tcl script for re-creating project 'artix_7_xdma'
+#
+# Generated by Vivado on Wed Feb 12 14:28:45 PST 2020
+# IP Build 2548770 on Fri May 24 18:01:18 MDT 2019
+#
+# This file contains the Vivado Tcl commands for re-creating the project to the state*
+# when this script was generated. In order to re-create the project, please source this
+# file in the Vivado Tcl Shell.
+#
+# * Note that the runs in the created project will be configured the same way as the
+#   original project, however they will not be launched automatically. To regenerate the
+#   run results please launch the synthesis/implementation runs as needed.
+#
+#*****************************************************************************************
+# NOTE: In order to use this script for source control purposes, please make sure that the
+#       following files are added to the source control system:-
+#
+# 1. This project restoration tcl script (bp_fpga.tcl) that was generated.
+#
+# 2. The following source(s) files that were local or imported into the original project.
+#    (Please see the '$orig_proj_dir' and '$origin_dir' variable setting below at the start of the script)
+#
+#    "/home/paul/artix_7_test/artix_7_xdma_dram_host_bp_cfg_new_exported/artix_7_xdma.srcs/pre-alpha-release/external/basejump_stl/bsg_misc/bsg_defines.v"
```

Comment:
I don't really know much about vivado.  Is it a problem that these paths are your local paths?

Reply:
These are comments in the tcl file (beginning with "#"), vivado will ignore all of them.